attribute_set/models/attribute_attribute.py

```diff
@@ -456,10 +472,10 @@ def write(self, vals):
 
     @api.multi
```

Comment:
Implicitly deprecated in v12 because invalid in v13
```suggestion
```

Reply:
Should I remove these `@api.multi` from everywhere in the code ? Or only here to override `unlink()` ?

Reply:
everywhere, yes